benchmarks/tool/compile_benchmarks.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2018, the Dart project authors.  Please see the AUTHORS file
```

Comment:
2022

Reply:
All of the files in this PR are new, should I update copyright years in all of the files?

Reply:
Updated copyright years in tool/.